styleguide/Styleguide.js

```diff
@@ -9,8 +9,6 @@ import Panels from './sections/Panels';
 
 import styles from './styleguide.css';
 
-import Icon from 'components/Icon/Icon';
```

Comment:
Wasn't being used so was throwing a linter error.

Need to add an Icon section at some point though.